x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/response/schemas.ts

```diff
@@ -45,6 +45,8 @@ export const output_index = t.string;
 export const saved_id = t.string;
 export const timeline_id = t.string;
 export const timeline_title = t.string;
+export const anomaly_threshold = PositiveInteger;
```

Comment:
Later we can change this to be more strict, like RiskScore codec type which ensures things are between two values such as 0 and 100. No changes needed. Just a pointer.